string table_name
---
bool table_existed
bool success
